.circleci/config.yml

```diff
@@ -32,93 +29,89 @@ commands:
       - restore_cache:
+          # First matched/found key wins and following keys are not tried
           keys:
-            #Restore existing cache for matching modules digest, validated to be exactly the same as in github current commit.
-            #This cache was made on top of below caches, if previously existing. If no module definition changed, we reuse this one. Otherwise...
-            - heads-modules-and-patches-{{ checksum "./tmpDir/all_modules_and_patches.sha256sums" }}{{ .Environment.CACHE_VERSION }}
-            #If precedent cache not found, restore cache for coreboot module (and patches) and musl-cross-make digests (coreboot: triannual release)
-            #Otehrwise....
-            - heads-coreboot-musl-cross-{{ checksum "./tmpDir/coreboot_musl-cross.sha256sums" }}{{ .Environment.CACHE_VERSION }}
-            #If precedent cache not found. Restore cache for musl-cross-make module digest (rarely modified).
-            #Otherwise, we build cleanly.
-            - heads-musl-cross-{{ checksum "./tmpDir/musl-cross.sha256sums" }}{{ .Environment.CACHE_VERSION }}
+            # Cache for matching modules digest, validated to be exactly the same as in github current commit.
+            # This cache was made on top of below caches, if previously existing.
+            # If no module definition changed, we reuse this one
+            - nix-docker-heads-modules-and-patches-{{ checksum "./tmpDir/all_modules_and_patches.sha256sums" }}{{ .Environment.CACHE_VERSION }}
+
+            # Cache for coreboot module (and patches) and musl-cross-make digests (coreboot: triannual release)
+            - nix-docker-heads-coreboot-musl-cross-{{ checksum "./tmpDir/coreboot_musl-cross.sha256sums" }}{{ .Environment.CACHE_VERSION }}
+
+            # Cache for musl-cross-make module digest (rarely modified).
+            - nix-docker-heads-musl-cross-{{ checksum "./tmpDir/musl-cross.sha256sums" }}{{ .Environment.CACHE_VERSION }}
       - run:
           name: Download and neuter xx20 ME (keep generated GBE and extracted IFD in tree)
           command: |
             ./blobs/xx20/download_parse_me.sh
-
       - run:
-          name: Download and neuter xx30 ME (keep generated GBE and extracted IFD in tree)
           # me_cleaner.py present under heads xx30 blobs dir comes from https://github.com/corna/me_cleaner/blob/43612a630c79f3bc6f2653bfe90dfe0b7b137e08/me_cleaner.py
+          name: Download and neuter xx30 ME (keep generated GBE and extracted IFD in tree)
           command: |
             ./blobs/xx30/download_clean_me_manually.sh -m $(readlink -f ./blobs/xx30/me_cleaner.py)
-
       - run:
           name: Download and extract t530 vbios roms for dgpu boards
           command: |
+            echo skipping for now
+            exit 0
             ./blobs/xx30/vbios_t530.sh
-
       - run:
           name: Download and extract w530 vbios roms for dgpu boards
           command: |
+            echo skipping for now
+            exit 0
```

Comment:
@JonathonHall-Purism haven't tested thanks for the pointer, since those boards moved to unmaintained.

BTW, @JonathonHall-Purism if you provide commit, I would gladly add back coreboot 4.11 based librem boards based on kgpe-d16 workstation under CirlceCI if provided?

Reply:
@JonathonHall-Purism I was unsuccessful fixing this quickly because VBIOS helpers are now outdated and gems and bundles are not happy in current state for those now unmaintained boards. Won't fix until dGPU board owners jump in the discussion if they care enough about re-upstreaming their boards.

TLDR: won't fix without collaboration

Reply:
In general I'd prefer not to leave broken stuff in master (especially when we have a branch containing the broken stuff to be fixed) so it's not a burden on maintenance.  Not asking to fix it :wink: 

This is not that much though, let's just try not to let it get out of control :grin: